autoblocks/_impl/testing/models.py

```diff
@@ -13,6 +31,26 @@ class Threshold:
     gte: Optional[float] = None
 
 
+@dataclasses.dataclass()
+class EventEvaluation:
+    evaluator_external_id: str
+    score: float
+    id: Optional[str] = dataclasses.field(default_factory=lambda: str(uuid.uuid4()))
+    metadata: Optional[dict] = None
+    threshold: Optional[Threshold] = None
+
+    @classmethod
+    def to_json(cls, event_evaluation):
+        return dict(
+            evaluatorExternalId=event_evaluation.evaluator_external_id,
+            id=str(event_evaluation.id),
+            score=event_evaluation.score,
+            metadata=dict(event_evaluation.metadata) if event_evaluation.metadata else None,
+            threshold=dict(event_evaluation.threshold) if event_evaluation.threshold else None,
+        )
+
+
+# TODO: Rename TestEvaluation?
```

Comment:
@nicolewhite

Reply:
why wouldn't these be the same thing?

Reply:
After reviewing the other related comments - 

I'll update with just extending the `Evaluation` class then working on obfuscating the `evaluatorExternalId` stuff

Reply:
why does it need to be extended? shouldn't it be the same thing?

Reply:
I just need to be able to add an optional `metadata` dict property here

Reply:
yeah we want that for both online and offline evaluations, so just update the current one

Reply:
Yup - bad wording on my part - by extend I meant update